src/App.js

```diff
@@ -10,19 +11,24 @@ import store from './_store';
 
 const history = createBrowserHistory();
 
-
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
I recommend changing this component to stateless instead of disabling the eslint rule. We are not passing it any prop so it can just be a stateless function